### ISSUE TITLE
Will the training set of Rain12600 be a test set of 1400 after 100 epochs?

### ISSUE THREAD

**Issue by FUNGERN** [csdwren/PReNet#17]
Hi, thank you for your contribution!
Will the training set of Rain12600 be a test set of 1400 after 100 epochs?
Looking forward to your reply, thanks.

**Comment by csdwren** [comment]
yes, it takes longer time. You can set larger step size when generating training patches to reduce training time, and I think the performance is comparable.
